cdist/conf/type/__rvm/explorer/state

```diff
@@ -28,7 +28,7 @@ if [ "$user" = "root" ]; then
         echo absent
     fi
 else
-    if su - $user -c "[ -d \"\$HOME/.rvm\" ]" ; then
+    if su - "$user" -c "[ -d '\$HOME/.rvm' ]" ; then
```

Comment:
This breaks the intended behaviour -- '$HOME' will not expand in the su-environment.

Reply:
reverted.